app/views/roasts/index.html.erb

```diff
@@ -0,0 +1,49 @@
+<%= render "shared/partials/nav-global" %>
+<%= render "shared/partials/header-global" %>
+
+<!-- first loop around main and second loop around roasts_brands section -->
+<!-- can loop the markup through Rails since each roast is templated -->
+
+<section class="roasts__slider-wrapper">
+    <div class="slider-axis">
+      <%= image_tag("light-slider-bean.svg", alt:"") %>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <%= image_tag("dark-slider-bean.svg", alt:"") %>
+    </div>
+    <div class="slider"><input type="range" value="0" min="0" max="8" step="1" class="slider__range"></div>
+</section>
+
+<main class = "roasts">
+  <% roasts.each_with_index do |roast, index| %>
+    <% roastIndex = index %>
+    <section class = "roasts__view" id = "roast<%= index %>">
+      <div class="view__image-wrapper">
+        <%= image_tag(roast.image.thumb('200x200#').url, alt:"") %>
+      </div>
+      <h2><%= roast.name %></h2>
+      <p><strong><%= roast.description %></strong></p>
+      <p><%= roast.sub_description %></p>
+    </section>
+    
+    <% if roast.roast_brands.any? %>
+      <% roast.roast_brands.each_with_index do |brand, index| %>
+        <section class="roasts__brands" id="roast<%= roastIndex %>brand<%= index %>">
+          <h2>Popular Brands</h2>
+          <div class="brand__wrapper">
+            <div class="wrapper__brand">
+              <%=image_tag brand.image.thumb('200x200#').url%>
+              <h3><%= brand.brand_name %></h3>
+              <p><%= brand.roast_name %></p>
+            </div>
+        </section>
```

Comment:
1. you are missing a closing `</div>` here

2. If you move lines 37-39 outside of the brands loop, then the closing tags for those lines also need to be moved outside of the loop (after the `<% end %>` on the next line)